apps/els_core/src/els_text.erl

```diff
@@ -65,3 +127,94 @@ line_starts(Text) ->
 pos(LineStarts, {LineNum, ColumnNum}) ->
   {LinePos, _} = lists:nth(LineNum, LineStarts),
   LinePos + ColumnNum.
+
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+apply_edits_single_line_test_() ->
```

Comment:
Seems like eunit tests aren't run by CI. Should I convert these to CT?

Reply:
Either that or add a `eunit` target here: https://github.com/erlang-ls/erlang_ls/blob/main/.github/workflows/build.yml#L44